core/storage/base_model/gae_models.py

```diff
@@ -909,9 +909,11 @@ def get_snapshot_id(cls, instance_id: str, version_number: int) -> str:
         return '%s%s%s' % (
             instance_id, VERSION_DELIMITER, version_number)
 
+    # We expect Mapping because we want to allow models that inherit
+    # from BaseModel as the values, if we used Dict this wouldn't be allowed.
     def _prepare_additional_models(self) -> Mapping[str, BaseModel]:
-        """Prepares additional models needed for the commit process. The dict
-        is empty by default  and this method should be overriden in models that
+        """Prepares additional models needed for the commit process. The return
+        value is empty dict and this method should be overriden in models that
```

Comment:
"overridden" is misspelled.

Add "default" before "return value".

Add "an" before "empty dict".

Change "and" to "; however,"

Reply:
Done.